pkg/storage/pebble_mvcc_scanner_test.go

```diff
@@ -294,3 +295,105 @@ func TestMVCCScanWithMemoryAccounting(t *testing.T) {
 		cleanup()
 	}
 }
+
+// TestMVCCScanWithMVCCValueHeaders tests that when the rawMVCCValues
+// option is given to pebbleMVCCScanner, the returned values can be
+// parsed using the extended encoding and the value header is
+// preserved.
+func TestMVCCScanWithMVCCValueHeaders(t *testing.T) {
```

Comment:
should this also test `mvccscanner.get()` in particular in the case in which the newest value is a deletion with extended encoding?

Reply:
Added a get-based test. I also added a case that makes sure we hit both the seek and iter path.